ang/test/civiawards/dashboard/directives/more-filters-dashboard-action-button.controller.spec.js

```diff
@@ -195,6 +195,32 @@
           expect($scope.isNotificationVisible()).toBe(true);
         });
       });
+
+      describe('when filters response yields no awards types ids', () => {
+        beforeEach(() => {
+          originalCrmAPIMock = crmApiMock;
```

Comment:
In this case storing and resetting `crmApiMock` is not necessary, because in Line 9, it will be reset for every unit test anyways. Resetting is only necessary when you are updating a global variable(like CRM), which not an angular component. As angular components are set in beforeEach section, they do not need special attention.